components/engine_traits/src/tablet.rs

```diff
@@ -149,7 +149,7 @@ pub trait TabletFactory<EK>: Send + Sync {
     /// Check if the tablet with specified path exists
     fn exists(&self, path: &Path) -> bool;
 
-    #[cfg(any(test, feature = "testexport"))]
+    #[cfg(feature = "testexport")]
```

Comment:
No need to change.

Reply:
`set_state_storage` is only used in test cluster to decouple the creation of engine and node. So test cfg is not needed I think as it will not be used in other places.